src/Xamarin.Android.NamingCustomAttributes/Android.App/ActivityAttribute.cs

```diff
@@ -0,0 +1,74 @@
+using System;
+
+#if !JCW_ONLY_TYPE_NAMES
+using Android.Content.PM;
+using Android.Views;
+#endif  // !JCW_ONLY_TYPE_NAMES
+
+namespace Android.App
+{
+
+	[Serializable]
+	[AttributeUsage (AttributeTargets.Class, 
+			AllowMultiple=false, 
+			Inherited=false)]
+#if !JCW_ONLY_TYPE_NAMES
```

Comment:
We shouldn't need `JCW_ONLY_TYPE_NAMES` anymore; that was a `Java.Interop`-ism, so that these types wouldn't be exported from `Java.Interop.Tools.JavaCallableWrappers.dll` (and other assemblies?).

Since the new `Java.Interop` partial types are *not* `public` (i.e. implicitly "internal" unless overridden in another `partial` class...), `JCW_ONLY_TYPE_NAMES` shouldn't be needed *anywhere* anymore.